apisix/plugins/request-id.lua

```diff
@@ -129,6 +129,7 @@ local function gen_data_machine(max_number)
                     if err4 then
                         snowflake_inited = nil
                         data_machine = nil
+                        timers.unregister_timer("plugin#request-id")
```

Comment:
Could we add a test to ensure the number of `unregister_timer` call is correct? For instance, via injecting some code.